### ISSUE TITLE
Problem flashing Cetus brushed board

### ISSUE THREAD

**Issue by islonina** [BETAFPV/BETAFPV.github.io#3]
I have a problem flashing board from Cetus brushed fc. Already check couple of different usb c cable. Push USB adapter connector, tried on usb 2.0 and 3.0 port on my pc. Drivers works fine, shows as COM port and the led on fc is white. If I need to provide any there information let me know. Its starts from no OSD. Brand new from the box never flown.

**Comment by wenzhicode** [comment]
Hi, islonina
What is your specific problem?

**Comment by Prorok666PL** [comment]
I have the same problem.
Is there any good software for this drone? My windows 10 does not recognize the device.

**Comment by kenken64** [comment]
Yes same problem for me. The board keep blinking red and the OSD disappear

**Comment by drew-man** [comment]
After I flashed my Cetus brushed FC I lost my OSD. All it says now is "disarm". I tried to flash the OSD but that didn't seem to help any.

**Comment by marianoforte** [comment]
I have a problem that I flashed my Cetus brushed board and now the LED keeps blinking in green but I can't bind it to my Literadio 2 SE. I have already checked that's in D8 mode and also tried to flash the FC with the Cetus_1.0.0, Cetus_1.0.1, Lite_V3_1.0.0 and Lite_V3_1.0.1 firmwares and none seems to work, the issue continues.
Edit: On my OSD it says "RX Loss" idk if it's a problem with the receiver, if the FC is broken and is not longer able to detect an Frsky D8 transmitter or what. Please help!

**Comment by mofox** [comment]
> After I flashed my Cetus brushed FC I lost my OSD. All it says now is "disarm". I tried to flash the OSD but that didn't seem to help any.

Which .bin file did you use? You should use the 1.0.1 version if you have the BRUSHED Cetus. 

I had the same issue on my Cetus Pro, where I flashed it with the 1.2.2 version and got the same problem. I had to flash it with the 1.1.2 version, because it is an older hardware version of the board.

**Comment by maxfield-allison** [comment]
> > After I flashed my Cetus brushed FC I lost my OSD. All it says now is "disarm". I tried to flash the OSD but that didn't seem to help any.
> 
> Which .bin file did you use? You should use the 1.0.1 version if you have the BRUSHED Cetus.
> 
> I had the same issue on my Cetus Pro, where I flashed it with the 1.2.2 version and got the same problem. I had to flash it with the 1.1.2 version, because it is an older hardware version of the board.

any way to tell the hw revision without flashing the wrong one?

**Comment by zxczzx** [comment]
> > After I flashed my Cetus brushed FC I lost my OSD. All it says now is "disarm". I tried to flash the OSD but that didn't seem to help any.
> 
> Which .bin file did you use? You should use the 1.0.1 version if you have the BRUSHED Cetus.
> 
> I had the same issue on my Cetus Pro, where I flashed it with the 1.2.2 version and got the same problem. I had to flash it with the 1.1.2 version, because it is an older hardware version of the board.

I've had the same problem but when I reflashed FC with 1.1.2 it didn't bring back my OSD. Any thoughts?

**Comment by thecoz420** [comment]
I had the same problem after flashing a brushed lite v3 board with Lite_v3_1.0.1.bin.
No OSD, I can connect to com3, but no settings to change.
Can you give me a step by step instruction on how to restore the original firmware please?
Thanks

**Comment by SergeySigov** [comment]
> After I flashed my Cetus brushed FC I lost my OSD. All it says now is "disarm". I tried to flash the OSD but that didn't seem to help any.

I had the same problem after flashing the Cetus FPV with the Betafpv Configurator. The OSD disappeared when flashing any of the available versions (1.0.0 and 1.0.1). The OSD appeared after flashing with the configure_V3 configurator, any of the Cetus_brush_2021-xx-xx firmwares.
